GoogleChromeLabs/ps-analysis-tool
# Use warning icon on cookie issues panel

## Description
This PR adds warning icon to sidebar item named `Cookie Issues` in CLI dashboard



## Relevant Technical Choices



## Testing Instructions
- Run CLI dashboard server.
- In the sidebar, items with Cookie Issues title will have warning icon.
- Click on icon, the item should be selected and icon's color should change.
- Try on both site and sitemap


## Additional Information:



## Screenshot/Screencast
<img width="202" alt="Screenshot 2024-04-11 at 16 15 23" src="https://github.com/GoogleChromeLabs/ps-analysis-tool/assets/58820001/9dbb6d8a-bc62-4339-8d08-9fb3b46f1710">
<img width="402" alt="Screenshot 2024-04-11 at 16 15 38" src="https://github.com/GoogleChromeLabs/ps-analysis-tool/assets/58820001/effefe40-0975-44de-a462-b2b1591f49ba">



---

## Checklist



- ~This code is covered by unit tests to verify that it works as intended.~



Fixes #
